README.md

```
@@ -527,6 +529,22 @@ MinIO did not have POSIX-like API support.
 
 MinIO had specific requirements on storage layout. It is not flexible to adjust capacity. In SeaweedFS, just start one volume server pointing to the master. That's all.
 
+[Back to TOC](#table-of-contents)
+
+### Compared to RustFS ###
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: atx; Actual: atx_closed

(MD003, heading-style)